sdk/search/search-documents/src/generated/data/models/index.ts

```diff
@@ -107,7 +108,7 @@ export interface SearchRequest {
   orderBy?: string;
   /**
    * A value that specifies the syntax of the search query. The default is 'simple'. Use 'full' if
-   * your query uses the Lucene query syntax. Possible values include: 'simple', 'full'
+   * your query uses the Lucene query syntax. Possible values include: 'Simple', 'Full'
```

Comment:
it still feels like a codegen bug that the possible list is capitalized. If a user passes in 'Simple' instead of 'simple', would it still work? (I know TS might complain, but what happens if they're just using JS?)

/cc @joheredi

Reply:
TS will complain. JS will allow it

Reply:
Thanks for bringing this up Jeff, yes this is definitely a bug in Autorest.Typescriopt (v5). @sarangan12 do you know if the swagger changed [here](https://github.com/Azure/azure-rest-api-specs/blob/33c52e4f87f3ae21611d45f34db65f9ccc510ea6/specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchindex.json#L197). Or were you fixing this by hand previously?

I can't find a recent change in Autorest.Typescript that may have regressed this. Anyway, I'll file a bug and make sure this is no happening in the new Autores.Typescript (v6)

Reply:
If JS allows it and the user sends it... does the server handle it correctly? If so, I have no problem here, otherwise I worry we'll get users into an error state they don't understand.